### ISSUE TITLE
Alt text does not work in <Draggable>

### ISSUE THREAD

**Issue by madebyisaacr** [framer/plugins#284]
**Describe the bug**
The altText parameter on the `<Draggable>` component for image alt text does not work.

**To Reproduce**
1. Create a new plugin with `npm create framer-plugin@latest`
2. Replace the contents of App.tsx with this code: 
```TSX
import { framer, Draggable } from "framer-plugin";
import "./App.css";

framer.showUI({
	position: "top right",
	width: 240,
	height: 95,
});

export function App() {
	const onClick = async () => {
		await framer.addImage({
			image: "https://framerusercontent.com/images/bNpMTgXS0nFU95Rn9bBQHjLFE.jpg",
			name: "Image.png",
			altText: "ALT TEXT",
		});
	};

	return (
		<main>
			<Draggable
				data={{
					type: "image",
					previewImage: "https://framerusercontent.com/images/bNpMTgXS0nFU95Rn9bBQHjLFE.jpg",
					image: "https://framerusercontent.com/images/bNpMTgXS0nFU95Rn9bBQHjLFE.jpg",
					altText: "ALT TEXT",
				}}
			>
				<button className="framer-button-primary" onClick={onClick}>
					Insert Image
				</button>
			</Draggable>
		</main>
	);
}

```
3. Click the button. The image is inserted with alt text.
4. Drag and drop the button. The image is inserted without alt text.

**Expected behavior**
When dragging and dropping an image, it should include the alt text set on `<Draggable>`.

I am certain that I am not using the wrong parameter, because it autocompletes `altText` and it's documented here: https://www.framer.com/developers/reference/plugins-drag-and-drop-dragdata
<img width="880" height="163" alt="Image" src="https://github.com/user-attachments/assets/ed2a0634-68ee-4c94-8d0d-539e9fe23a1f" />

**Video**
https://youtu.be/s6H0q31gg4s

**Additional context**
Discovered this bug while adding alt text to Unsplash: https://github.com/framer/plugins/pull/283

**Comment by niekert** [comment]
Thanks! This is fixed on beta. Will close the issue when it goes to stable.
